src/main/java/bern/functionalities/Ui.java

```diff
@@ -0,0 +1,276 @@
+     * @return The reply by the bot.
+     * @throws BernException If input is invalid.
+     */
+    public String ifList(String input, ArrayList<Task> arListTask) throws BernException {
+        String result = "";
+
+        for (int i = 0; i < arListTask.size(); i++) {
+            result += String.valueOf(i + 1)
+                    + ". "
+                    + arListTask.get(i).toString()
+                    + (i == arListTask.size() - 1 ? "" : "\n");
+        }
+        if (arListTask.size() == 0) {
+            result = "There is no task.";
+        }
+
+        return result;
+    }
+
+    /**
+     * A method to get the reply if input starts with delete.
+     *
+     * @param input The given command.
+     * @param arListTask The initial ArrayList of Tasks.
+     * @return The reply by the bot.
+     * @throws BernException If input is invalid.
+     */
+    public String ifDelete(String input, ArrayList<Task> arListTask) throws BernException {
+        if (input.length() == 6 || (input.length() == 7 && input.substring(6, 7).equals(" "))) {
+            throw new EmptyDescriptionException("done");
+        } else if (!new Parser().isANumber(input.substring(7))) {
+            throw new InvalidCommandException(input);
+        } else if (Integer.parseInt(input.substring(7)) > arListTask.size()) {
+            throw new IndexException(input);
+        }
+        int index = Integer.parseInt(input.substring(7)) - 1;
+        String temp = arListTask.get(index).toString();
+        new TaskList().removeTask(index, arListTask);
+        return "Noted! I've removed this task:\n"
+                + temp + "\n"
+                + "Now you have " + String.valueOf(arListTask.size())
+                + (arListTask.size() == 1 ? " task in the list" : " tasks in the list");
+    }
+
+    /**
+     * A method to get the reply for any command.
+     *
+     * @param c The command type as delineated by enum class Command.
+     * @param input The given command.
+     * @param arListTask The initial ArrayList of Tasks.
+     * @return The reply by the bot.
+     * @throws BernException If input is invalid.
+     */
+    public String getReply(Command c, String input, ArrayList<Task> arListTask) throws BernException {
+        switch (c) {
+        case DONE:
+            return new Ui().ifDone(input, arListTask);
+        case DEADLINE:
+            return ifDeadline(input, arListTask);
+        case EVENT:
+            return ifEvent(input, arListTask);
+        case TODO:
+            return ifToDo(input, arListTask);
+        case BYE:
+            return ifBye(input, arListTask);
+        case FIND:
+            return ifFind(input, arListTask);
+        case LIST:
+            return ifList(input, arListTask);
+        case DELETE:
+            return ifDelete(input, arListTask);
+        case INVALID:
+            throw new InvalidCommandException(input);
+        default:
+            return "";
```

Comment:
This might just be my preference but I think long switch cases might make code less readable.
Maybe consider the usage of `HashTable<Command, Function>` ? Although this is probably down to individual preference.